src/samples/ChildBenefitsClaim/cookiePage/index.tsx

```diff
@@ -5,18 +5,28 @@ import AppFooter from '../../../components/AppComponents/AppFooter';
 import CookiePageTable from './CookiePageTable';
 import { scrollToTop } from '../../../components/helpers/utils';
 import MainWrapper from '../../../components/BaseComponents/MainWrapper';
+import { useLocation } from 'react-router-dom';
+import { registerServiceName } from '../../../components/helpers/setPageTitleHelpers';
 
 const FIND_OUT_MORE_URL = 'https://www.tax.service.gov.uk/help/cookie-details';
 
 export default function CookiePage() {
   const { t } = useTranslation();
+  const location = useLocation();
   scrollToTop();
 
+  const searchParams = new URLSearchParams(location.search);
+  const appNameHeader = searchParams.get('appname');
+  const serviceNamePageTitle = searchParams.get('serviceNamePageTitle');
+  const serviceParam = searchParams.get('serviceParam');
+  
+  registerServiceName(t(serviceNamePageTitle));
```

Comment:
If we use the React context idea, we could remove these

Reply:
We don't have context availability in cookiePage. It's falling out of the context so we could not update the context values.

Best way to solve this is to have a common context at 'TopLevel' which is not there atm. I can try moving existing HMRCcontext which is at Assignment level instead of creating new